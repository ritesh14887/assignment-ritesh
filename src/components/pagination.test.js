import React from 'react';
import renderer from 'react-test-renderer';
import Pagination from './pagination';

test('Pagination renders correctly', () => {
  const pageData = {
    pages: 2,
  };
  const tree = renderer
    .create(<Pagination pagedata={pageData} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
