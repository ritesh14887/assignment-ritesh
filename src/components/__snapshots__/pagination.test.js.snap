// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pagination renders correctly 1`] = `
<div
  className="Pagination-pagination-0-2-1"
>
  <button
    className="page-link"
    onClick={[Function]}
    type="button"
    value={1}
  >
    1
  </button>
  <button
    className="page-link"
    onClick={[Function]}
    type="button"
    value={2}
  >
    2
  </button>
</div>
`;
